src-tauri/src/main.rs

```diff
@@ -3,12 +3,38 @@
     windows_subsystem = "windows"
 )]
 
-use std::sync::Mutex;
+use std::{fs, sync::Mutex};
 
 use dropbox_sdk::oauth2;
 
 const APPKEY: &str = "z4c46xuhvi38jhh";
 
+#[derive(serde::Serialize)]
+struct StringFromErr(String);
+impl<E> From<E> for StringFromErr
+where
+    E: std::error::Error,
+{
+    fn from(e: E) -> Self {
+        Self(format!("{}", e))
+    }
+}
+
+#[derive(serde::Deserialize, std::fmt::Debug)]
+struct Template {
+    /// Display name for the template. Template names can be up to 256 bytes.
```

Comment:
I'm not a rust developer, does a triple `/` means something different than a double?